### DISCUSSION TITLE
How to remove that V-line from the panel?

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#709] [Questions] [answered]
When using the [B00merang Windows 8 Metro theme](https://github.com/B00merang-Project/Windows-8-Metro) I get that line between the Linux Mint logo and Opera:

![image](https://github.com/user-attachments/assets/8790e249-0d95-4522-ab37-b0808f671bc3)

Where in the cascading style sheet do I remove that? I've been able to remove one of the lines from `.applet-separator`, but still remained one.

**Comment by anaximeno** [comment, marked as answer]
if you're on cinnamon just right click it then click remove.
